presto-hive-hadoop2/bin/run_hive_alluxio_tests.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+set -euo pipefail -x
+
+. "${BASH_SOURCE%/*}/common.sh"
+
+source presto-product-tests/conf/product-tests-config-hdp3.sh
+
+ALLUXIO_DOCKER_COMPOSE_LOCATION="${INTEGRATION_TESTS_ROOT}/conf/alluxio-docker.yml"
+
+function start_alluxio_containers() {
+  # stop already running containers
+  docker-compose -f "${ALLUXIO_DOCKER_COMPOSE_LOCATION}" down || true
+
+  # catch terminate signals
+  trap termination_handler INT TERM
```

Comment:
function should not set `trap` -- strongly recommended, since this overrides previous traps.

btw why do you define `start_alluxio_containers` as a function?
it's called once & not reused.
IMO the script would read more easily, top-down, without the functions (except those that are reused)